packages/Spinner/Spinner.md

```diff
@@ -1,28 +1,220 @@
+### Usage criteria
+
+- Avoid using a spinner whenever possible. Make content available to the user as soon as possible
+- User feedback should be present when a user action takes longer than 0.1 seconds
+- A Spinner is most appropriate for providing feedback for actions that last between 2 and 10 seconds
+- Actions that take longer than 10 seconds ideally provide feedback with relative progress, such as a progress bar
+- Recommend not to use for full page loading
+- Label should be short and concise. Spinner label should accurately explain the state of the requested action along with relevant items being loaded, “Loading page content”, “Logging in”, “Processing payment”
+- Should use for asynchronous requests that take between 1 to 4 seconds to load
+- Use the small spinner when applied on a button or other small interactive elements such as Toggles or links
+- Use the large spinner when affecting an entire web page or content block, whose content has not finished loading, such as paginated content
+- When overlaying buttons, the spinner’s colour should match the button’s colour (green spinner for green buttons, and purple spinner for purple buttons)
+
```

Comment:
Add two more bullet points:
* Both the small and large spinner have an overlay background to visually indicate content behind it is inaccessible
* Use the standalone spinner (small/large) as an interstitial placeholder for loading content